modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Numeric/Numeric.tsx

```diff
@@ -205,13 +193,24 @@ const Numeric: React.FC<IProps> = ({
 	}, [inputMask, symbolsProp]);
 
 	const inputValue = useMemo<IMaskedNumber>(() => {
-		const newValue =
+		let newValue =
 			((localizedValue?.[editingLanguageId] ??
 				localizedValue?.[defaultLanguageId]) ||
 				value) ??
 			predefinedValue ??
 			'';
 
+		if (dataType === 'double') {
+			const symbolsValue = newValue.match(NON_NUMERIC_REGEX);
+
+			if (symbolsValue && !newValue.includes(symbols.decimalSymbol)) {
```

Comment:
To be honest, I didn’t want to change the existing code and cause a bug 🙈. 

But I can change to keep consistency.

Reply:
Because of Typescript I'm getting the following error:

<img width="813" alt="Captura de Tela 2021-12-23 às 14 04 39" src="https://user-images.githubusercontent.com/34186632/147271643-2eea15c4-b1f4-440b-8626-eac7711f6c09.png">

That's way I'm going to change, in both cases, to:

```
parameter = decimalSymbol ? parameter.replace(decimalSymbol[0], '.') : parameter;
```

Reply:
Haha. No problem. Regardless, I think that your new approach is better, so we don't have to call the `replace` even if the `decimalSymbol` is undefined.